konduit-serving-pipeline/src/main/java/ai/konduit/serving/pipeline/impl/step/bbox/filter/BoundingBoxFilterStepRunner.java

```diff
@@ -0,0 +1,53 @@
+package ai.konduit.serving.pipeline.impl.step.bbox.filter;
+
+import ai.konduit.serving.pipeline.api.context.Context;
+import ai.konduit.serving.pipeline.api.data.BoundingBox;
+import ai.konduit.serving.pipeline.api.data.Data;
+import ai.konduit.serving.pipeline.api.data.ValueType;
+import ai.konduit.serving.pipeline.api.step.PipelineStep;
+import ai.konduit.serving.pipeline.api.step.PipelineStepRunner;
+import lombok.AllArgsConstructor;
+
+import java.util.Arrays;
+import java.util.List;
+import java.util.stream.Collectors;
+
+@AllArgsConstructor
+public class BoundingBoxFilterStepRunner implements PipelineStepRunner {
+
+    protected final BoundingBoxFilterStep step;
+
+    @Override
+    public void close() {
+
+    }
+
+    @Override
+    public PipelineStep getPipelineStep() {
+        return step;
+    }
+
+
+    @Override
+    public Data exec(Context ctx, Data data) {
+
+        String[] classesToKeep = step.classesToKeep();
+        List<BoundingBox> boundingBoxes = data
+                .getListBoundingBox("img_bbox")
```

Comment:
Don't hardcode the key, this should be specified in the configuration (BoundingBoxFilterStep)